py-data-aggregation/db/aggregation_consumer.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import os
+import sys
+from mpi4py import MPI
+
```

Comment:
Should we add some quick comments to these producer and consumer python scripts just for when another dev comes back to maintain them?